integrations/symfony-bundle.rst

```diff
@@ -244,6 +244,32 @@ services are:
                     my_client:
                         service: 'my_custom_client_service'
 
+    .. versionadded:: 1.17
```

Comment:
Not sure about the version here, what do you think?

Reply:
yep, that version sounds right

Reply:
Thank you, @dbu!